requirements.txt

```diff
@@ -0,0 +1,25 @@
+Automat==0.6.0
+Twisted==17.5.0
+ZooKeeper==0.4
+argparse==1.2.1
+asn1crypto==1.0.1
+attrs==19.1.0
+cffi==1.10.0
+constantly==15.1.0
+cryptography==2.3
+enum34==1.1.6
+hyperlink==18.0.0
+idna==2.8
+incremental==17.5.0
+ipaddress==1.0.22
+pyOpenSSL==19.0.0
+pyasn1==0.4.5
+pyasn1-modules==0.2.5
+pycparser==2.19
+pycrypto==2.6.1
+rollingpin==1.0
```

Comment:
should probably drop this line -- hard to install itself!

Reply:
haha, ty